tests/test_pc.py

```diff
@@ -203,3 +206,154 @@ def test_git_irk_equivalence(quad_scheme, order, stage_type):
         stepper.advance()
         assert numpy.allclose(stepper.solver.snes.ksp.computeEigenvalues(), 1.0)
         t.assign(float(t) + float(dt))
+
+
+@pytest.mark.parametrize('family', (ContinuousPetrovGalerkinScheme, RadauIIA))
```

Comment:
We need some kind of test criterion, that we get the correct answer or have a low iteration count or some such.

Reply:
I set `-ksp_max_it 15` and added tests for the eigenvalues